src/modules/app/StateType.js

```diff
@@ -10,6 +10,7 @@ import DisclaimerModel from '../endpoint/models/DisclaimerModel'
 import SprungbrettJobModel from '../endpoint/models/SprungbrettJobModel'
 import WohnenOfferModel from '../endpoint/models/WohnenOfferModel'
 import type { LocationState } from 'redux-first-router'
+import PointOfInterestModel from '../endpoint/models/PointOfInterestModel'
```

Comment:
Should we go with POI instead of PointOfInterrest? Way shorter and has a more specific meaning because its a well known abbreviation.

Reply:
It doesn't have a "more specific meaning" - it's literally the same meaning afaik - but I like POI better because it's shorter and its meaning is clear

Reply:
The same meaning but you see it more often as POI instead of Point Of Interrest.